Library/Homebrew/dev-cmd/generate-formula-api.rb

```diff
@@ -55,6 +55,13 @@ def generate_formula_api
       tap_migrations_json = JSON.dump(tap.tap_migrations)
       File.write("api/formula_tap_migrations.json", tap_migrations_json) unless args.dry_run?
 
+      internal_formula_v3_hash = {
+        "aliases"        => tap.alias_table,
+        "formulae"       => {},
+        "renames"        => tap.formula_renames,
+        "tap_migrations" => tap.tap_migrations,
+      }
```

Comment:
This is meant to match the formula cache in the `api/formula.rb` file.

https://github.com/Homebrew/brew/blob/36024e2a4db7127114c39bbdb62a812816e6a0bf/Library/Homebrew/api/formula.rb#L45-L47